### PR TITLE
updated the Holmes-Storage documentation

### PR DESCRIPTION
**Pull request by cli0**
I added more details and references to the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,32 +4,65 @@
...
+Holmes-Storage supports multiple databases and splits them into two categories: Object Stores and Document Stores. 
+Object Stores are responsible for storing the file-based malicious objects collected by the analyst: binary files such as PE32 and ELF, PDFs, HTML code, Zips files etc. Document Stores are contain the results of the Holmes-Totem and Holmes-Totem-Dynamic analytic results of the files in Object Stores. This was done to enable users to more easily select their preferred solutions while also allowing the mixing of databases for optimization purposes. In production environments, we strongly recommend using an [S3](https://aws.amazon.com/documentation/s3/) compatible Object Store, such as [RIAK-CS](http://docs.basho.com/riak/cs/latest), and a clustered deployment of [Cassandra](http://cassandra.apache.org/) for the Document Stores.
 
-#### Object Stores
-We support three primary object storage databases. We recommend S3 for large deployments.
+#### I. Object Stores
+We support three primary object storage databases. 
 - Local File System
```
Comment (webstergd) [inline README.md]: we dropped local file system in version 2 of storage. given mino is so easy to install I don't see a use case for adding it back anytime soon. mind removing this bullet please

File: README.md [hunk 2]
```diff
@@ -4,32 +4,65 @@
...
+Holmes-Storage supports multiple databases and splits them into two categories: Object Stores and Document Stores. 
+Object Stores are responsible for storing the file-based malicious objects collected by the analyst: binary files such as PE32 and ELF, PDFs, HTML code, Zips files etc. Document Stores are contain the results of the Holmes-Totem and Holmes-Totem-Dynamic analytic results of the files in Object Stores. This was done to enable users to more easily select their preferred solutions while also allowing the mixing of databases for optimization purposes. In production environments, we strongly recommend using an [S3](https://aws.amazon.com/documentation/s3/) compatible Object Store, such as [RIAK-CS](http://docs.basho.com/riak/cs/latest), and a clustered deployment of [Cassandra](http://cassandra.apache.org/) for the Document Stores.
```
Comment (webstergd) [inline README.md]: Document Stores are contain => Document Stores contain 

Document Stores are contain the results of the Holmes-Totem and Holmes-Totem-Dynamic analytic results of the files in Object Stores. => Document Stores contain the output of Holmes-Totem and Holmes-Totem-Dynamic Services.

File: README.md [hunk 3]
```diff
@@ -4,32 +4,65 @@
...
+We support three primary object storage databases. 
 - Local File System
 - S3 compatible
 - (Soon) MongoDB Gridfs
 
-#### Document Stores
-We support two primary object storage databases. We recommend a Cassandra cluster for large deployments.
+There are several tools you can use for implementing Object Stores. Depending on the intended scale of your work with Holmes-Storage, we would recommend:
+
+- [AWS](https://aws.amazon.com/) for cloud computing, industry and large-scale implementation
```
Comment (webstergd) [inline README.md]: might be good to have this as a table. Maybe we could have columns  for workstation, midsized, and large-scale

File: README.md [hunk 4]
```diff
@@ -4,32 +4,65 @@
...
+#### II. Document Stores
+We support two primary object storage databases. 
 - Cassandra
 - MongoDB
 
+We recommend a Cassandra cluster for large deployments.
+
 ##### Cassandra 
-Holmes-Storage supports a single or cluster installation of Cassandra version 3.5.x and higher. The version requirement is because of the significant improvement in system performance when leveraging the newly introduced [SASIIndex](https://github.com/apache/cassandra/blob/trunk/doc/SASI.md) for secondary indexing. We highly recommend deploying Cassandra as a cluster with a minimum of three Cassandra nodes in production environments.
+Holmes-Storage supports single node or cluster installation of Cassandra version 3.5.x and higher. The version requirement is because of the significant improvement in system performance when leveraging the newly introduced [SASIIndex](https://github.com/apache/cassandra/blob/trunk/doc/SASI.md) for secondary indexing. We highly recommend deploying Cassandra as a cluster with a minimum of three Cassandra nodes in production environments.
```
Comment (webstergd) [inline README.md]: we should bump this up to 3.10. Cassandra did a lot of bug fixes for sasi and material views. 3.5 works...3.10 works a lot better.

leveraging the newly introduced [SASIIndex](https://github.com/apache/cassandra/blob/trunk/doc/SASI.md) for secondary indexing. => leveraging the newly introduced [SASIIndex](https://github.com/apache/cassandra/blob/trunk/doc/SASI.md) and Material Views [need a link]

**Review by webstergd — CHANGES_REQUESTED** [review]
looks awesome. Thanks @cli0 

I made some suggestions / quick fixes

**Comment by cynexit** [comment]
Nice job, I agree with Georges changes. We might bring back local-fs later but for now it's definitely gone.

**Review by webstergd — APPROVED** [review]

**Comment by ms-xy** [comment]
@cynexit regarding local-fs is it incompatible with changes or just nobody that updated it? I do use it all the time for testing :P

**Pull request merged**